<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 222, 78);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>871</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
font: 35pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>Cat Talk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_name">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>530</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>박의용</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>IP 주소</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_ip">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>600</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>10.10.21.113</string>
      </property>
     </widget>
     <widget class="QPushButton" name="entrance_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>750</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 20px;
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)
}</string>
      </property>
      <property name="text">
       <string>프로필 선택</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>유저 이름</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="profile_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>896</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="pixmap">
       <pixmap>gb.jpg</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_port">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>670</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>5456</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>670</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>포트 번호</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="profile_choice_label">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>670</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>profile_label</zorder>
     <zorder>label_4</zorder>
     <zorder>user_name</zorder>
     <zorder>label_6</zorder>
     <zorder>user_ip</zorder>
     <zorder>entrance_btn</zorder>
     <zorder>label_5</zorder>
     <zorder>user_port</zorder>
     <zorder>label_7</zorder>
     <zorder>profile_choice_label</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="onechat_title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>접속유저</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="user_tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>781</width>
        <height>611</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
font: 15pt &quot;Cambria&quot;;</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>259</number>
      </attribute>
      <column>
       <property name="text">
        <string>프로필</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>접속여부</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="onechat_title_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>프로필</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="profile_id_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>접속 ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="id_label">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);

font: 25pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
font: 35pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>Cat Talk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="onechat_title_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>Me(나)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="msg_history_btn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)
}</string>
      </property>
      <property name="text">
       <string>지난 대화내용 조회</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 3</string>
     </attribute>
     <widget class="QTextBrowser" name="userList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
font: 15pt &quot;Cambria&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="msg_lienEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>660</y>
        <width>461</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>메세지를 작성 해 주세요</string>
      </property>
     </widget>
     <widget class="QPushButton" name="msg_send_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>670</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)
}</string>
      </property>
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chat_exit_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>740</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)
}</string>
      </property>
      <property name="text">
       <string>채팅방 나가기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>대화 상대</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="chat">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>381</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
font: 15pt &quot;Cambria&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="onechat_title_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>친구</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="chat_tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>341</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
font: 15pt &quot;Cambria&quot;;</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>259</number>
      </attribute>
      <column>
       <property name="text">
        <string>프로필</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>접속여부</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="onechat_title_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 25pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>대화 내용</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="friend_find_btn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>740</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)
}</string>
      </property>
      <property name="text">
       <string>친구 찾기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>750</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
font: 35pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>Cat Talk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 4</string>
     </attribute>
     <widget class="QTableWidget" name="history_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>731</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="message_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>731</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Message</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
font: 35pt &quot;Cambria&quot;;
color: rgb(0, 170, 255)</string>
      </property>
      <property name="text">
       <string>Cat Talk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>451</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>조회하실 아이디 또는 날짜를 클릭해주세요.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
